docs/source/changelog.rst

```diff
@@ -3,6 +3,33 @@
 Changes in Jupyter Qt console
 =============================
 
+.. _4.4:
+
+`4.4 on GitHub <https://github.com/jupyter/qtconsole/milestones/4.4>`__
+
+Additions
+~~~~~~~~~
+- :kbd:`Control-D` enters an EOT character if kernel is executing and input is
+  empty.
+- Implement block indent on multiline selection with :kbd:`Tab`.
+- Change the syntax highlighting style used in the console on the fly. It can
+  be done in the menu ``View > Syntax Style``.
+
+Changes
+~~~~~~~
+- Change :kbd:`Control-Shift-A` to select cell contents first.
+- Change default tab width to 4 spaces.
```

Comment:
Assuming to better accommodate Python.

Reply:
I really don't know, I didn't do that change, sorry.